app/assets/stylesheets/framework/form.scss

```diff
@@ -0,0 +1,44 @@
+// cleaning bootstrap. Can be removed once bootstrap will be stripped out
+label {
+  margin-bottom: 0;
+}
+// end cleaning bootstrap
+
+button,
+input,
+select,
+textarea {
+  // Fix dirty box model rendering on inputs
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  border: solid 1px $input-border;
+  background-color: $input-bg;
+  padding: .4rem;
+  border-radius: $radius;
+  font-size: 1rem;
+}
+
+[type=text],
+[type=email],
+[type=password] {
+  width: 320px;
+  box-shadow: inset 1px 0 1px #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.